src/io/flutter/view/InspectorPanel.java

```diff
@@ -539,6 +547,91 @@ public void showProperties(DiagnosticsNode diagnostic) {
     }
   }
 
+  private static SimpleTextAttributes textAttributesForLevel(DiagnosticLevel level) {
+    switch (level) {
+      case hidden:
+      case fine:
+        return SimpleTextAttributes.GRAYED_ATTRIBUTES;
+      case warning:
+        // TODO(jacobr): would be nice to use a yellow color for level warning.
```

Comment:
👍 

Maybe just cache one in this class?


    new SimpleTextAttributes(STYLE_PLAIN, Color.ORANGE);

Reply:
Good idea. Color.ORANGE works fine for both Dracula and the Default theme. Added.